tests/e2e/e2e_setup_test.go

```diff
@@ -340,6 +349,94 @@ func (s *IntegrationTestSuite) runValidators(c *chain, portOffset int) {
 	)
 }
 
+func (s *IntegrationTestSuite) runIBCRelayer() {
+	s.T().Log("starting Hermes relayer container...")
+
+	tmpDir, err := ioutil.TempDir("", "gaia-e2e-testnet-hermes-")
+	s.Require().NoError(err)
+	s.tmpDirs = append(s.tmpDirs, tmpDir)
+
+	gaiaAVal := s.chainA.validators[0]
+	gaiaBVal := s.chainB.validators[0]
+	hermesCfgPath := path.Join(tmpDir, "hermes")
+
+	s.Require().NoError(os.MkdirAll(hermesCfgPath, 0755))
+	_, err = copyFile(
+		filepath.Join("./scripts/", "hermes_bootstrap.sh"),
+		filepath.Join(hermesCfgPath, "hermes_bootstrap.sh"),
+	)
+	s.Require().NoError(err)
+
+	s.hermesResource, err = s.dkrPool.RunWithOptions(
+		&dockertest.RunOptions{
+			Name:       fmt.Sprintf("%s-%s-relayer", s.chainA.id, s.chainB.id),
+			Repository: "ghcr.io/cosmos/hermes-e2e",
```

Comment:
@nikever do you know how to add this as an image to our repo? I'm not positive of the process

Reply:
To avoid blocking this PR, let's create an issue to address this separately? 

It should be fine to use the `cosmos` image temporarily, The main concern about depending on external images, especially with the latest tag, is that whenever they make a change, our tests might get broken

Reply:
The credentials for the `osmobot` Docker account are saved as GitHub Secrets. You can use them to build the Image via CI as in https://github.com/osmosis-labs/osmosis/blob/main/.github/workflows/docker.yml#L24-L29

I will also create personal accounts for both of you on our Docker account.

Lets use a new repository `osmolabs/osmosis-e2e` for all these new images